web/tools/webpack.config.ts

```diff
@@ -171,6 +173,10 @@ const createConfig = (
           config: buildConfig
         }
       }),
+      new PreloadWebpackPlugin({
```

Comment:
Which resources are preloaded by adding this plugin? Is there a benefit without current setup?

Reply:
I would only add it if there is a realworld benefit because webpack configs get messy quickly

Reply:
main and runtime JS-Files get preloaded. I mean its hard to measure to be honest.
Some blogs recommend doing that but i'm also fine with reverting it.